python/docs/source/index.rst

```diff
@@ -72,20 +76,25 @@ DataFrames, Structured Streaming, Machine Learning (MLlib) and Spark Core.
           :alt: Spark Core and RDDs
      -
 
-.. _Index Page - Spark SQL and DataFrames:
+.. _Index Page - DataFrame:
 
-**Spark SQL and DataFrames**
+**DataFrame**
```

Comment:
This might be a stupid question since I'm not familiar enough with English expressions: why do we express `DataFrame` in plural?

Btw, I just follow the Pandas documentation (of course we don't necessarily have to follow them, though):

<img width="764" alt="Screenshot 2023-12-08 at 2 10 27 PM" src="https://github.com/apache/spark/assets/44108233/eb35dd5c-ac6e-45a2-9102-f8f4992fa276">

Reply:
Sounds reasonable to me. Thanks!